src/discussions/comments/comment/CommentEditor.jsx

```diff
@@ -142,6 +145,11 @@ CommentEditor.propTypes = {
   }).isRequired,
   onCloseEditor: PropTypes.func.isRequired,
   intl: intlShape.isRequired,
+  edit: PropTypes.bool,
+};
+
+CommentEditor.defaultProps = {
+  edit: true,
```

Comment:
can we make default edit: false , and pass edit prop only when we need it to be true?

Reply:
This component's primary purpose is editing and is always used for it. This component is also used in many components for editing propose. So default behavior should be editing true.  Only two use cases are not adding the edit reason.

Reply:
right.